include/zephyr/net/net_pkt_filter.h

```diff
@@ -242,6 +243,14 @@ bool npf_remove_all_rules(struct npf_rule_list *rules);
 		.tests = { FOR_EACH(Z_NPF_TEST_ADDR, (,), __VA_ARGS__) }, \
 	}
 
+#define NPF_PRIORITY(_name, _priority, ...)                                                        \
```

Comment:
A minimal example of how to create such priority rule could possibly be added to the pkt filter documentation (https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/connectivity/networking/api/net_pkt_filter.rst).